packages/rspack/src/loader-runner/index.ts

```diff
@@ -1101,6 +1101,12 @@ export async function runLoaders(
 			id2: resource
 		}
 	});
+
+	if (compiler.options.experiments.cache && compiler.options?.cache) {
+		// Serialize buildInfo to the Rust side for later persistent caching by Rust.
+		context._module.buildInfo.serialize();
```

Comment:
Maybe we could rename it to `sync`? All data in buildinfo or other fields should be added on the rust side to ensure data consistency, not just for persistent cache.

Reply:
rename to syncCustomFields